📝 Add repo for TS GitHub teams management (#277)
ministryofjustice/cloud-operations: README.md

|-|-|
| [GitHub actions](https://github.com/ministryofjustice/cloud-operations-github-actions) | Central configuration repository for [GitHub Actions Workflows](https://docs.github.com/en/actions/using-workflows/about-workflows), which can be called from our other repositories. |
| [Shared services infrastructure](https://github.com/ministryofjustice/staff-device-shared-services-infrastructure) | Creates infrastructure in the shared services account, including continuous integration and delivery pipelines |

> | [Technology Services GitHub Teams](https://github.com/ministryofjustice/staff-technology-services-github-teams) | Terraform to define and maintain a GitHub teams for the Technology Services. |

Reviewer: ```suggestion
| [Technology Services GitHub Teams](https://github.com/ministryofjustice/staff-technology-services-github-teams) | Terraform repository to define and maintain GitHub Teams for Technology Services. |
```
